cypress/e2e/page/categorization-pagetests.cy.js

```diff
@@ -0,0 +1,292 @@
+            cy.get("@config").then(p_config => {
+
+                cy.get("@dataDictionary").then(p_dataDictionary => {
+
+                    cy.dispatchToNuxtStore("saveDataDictionary", {
+
+                        data: p_dataDictionary,
+                        filename: p_config.data_dictionary,
+                        fileType: "json"
+                    });
+                });
+            });
+
+            // 4. Move to categorization page
+            // NOTE: Routing to the page prevents the Vuex store from being wiped
+            // when a page is 'visited' by Cypress
+            cy.window().its("$nuxt.$router").then(router => {
+
+                router.push({ path: "/categorization" });
+            });
+        });
+
+        // Description of task: User selects one non-tool group category and paints one column
+        // Expected results: Annotation nav and Next page button are enabled
+        it("simple categorization", () => {
+
+            // Action 1
+
+            // Select subject ID category
+            cy.get("[data-cy='categorization-table']")
+                .contains("Subject ID")
+                .click();
+
+            // Assert 1 - Nav and next button should not yet be enabled
+            cy.get("[data-cy='menu-item-annotation'] a")
+                .should("have.class", "disabled");
+            cy.get("[data-cy='button-nextpage']")
+                .should("have.class", "disabled");
+
+            // Action 2
+
+            // Link the first column in the column select table with the subject ID category
+            cy.get("[data-cy='column-linking-table'] tbody tr td")
+                .contains("participant_id")
+                .click();
+
+            // Assert 2 - Nav and next button should be enabled
+            // Assert 1
+            cy.get("[data-cy='menu-item-annotation'] a")
+                .should("not.have.class", "disabled");
+            cy.get("[data-cy='button-nextpage']")
+                .should("not.have.class", "disabled");
+        });
+
+        // Description of task: User selects multiple non-tool group category and paints multiple columns
+        // Expected results: Annotation nav and Next page button are enabled
+        it("multiple categorization", () => {
+
+            // Action 1
+
+            // Select 'Subject ID' category
+            cy.get("[data-cy='categorization-table']")
+                .contains("Subject ID")
+                .click();
+
+            // Assert 1 - Nav and next button should not yet be enabled
+            cy.get("[data-cy='menu-item-annotation'] a")
+                .should("have.class", "disabled");
+            cy.get("[data-cy='button-nextpage']")
+                .should("have.class", "disabled");
+
+            // Action 2
+
+            // Link the first column in the column select table with the 'Subject ID' category
+            cy.get("[data-cy='column-linking-table'] tbody tr td")
+                .contains("participant_id")
```

Comment:
Looking into this turned out to be more interesting than I realized it would be.

Firstly, agreed about the genericization re: the specific string. And for the simple categorization test that's just fine – with one exception (see below). In fact, that's how I first implemented this and other table row selects. You call `find("tr")` and use the `eq` function to select the nth index. 
What I've noticed is that sometimes `eq` can select an index you don't expect. I noticed this on the linking table so I just switched all table row selects to use the select by text method. There is likely a reason for this unexpected behavior. Either it's something we're doing (i.e. not specifying a row index correctly) or something with Vue/Cypress mixed.

The "interesting" part is that your pointing this surfaces something we haven't smoothed out since we were developing an app prototype and not something with more generic logic. Within the categorization pages `nextPageAccessible` we have the following code:

```
                // 4. Make sure one (and only one) column has been categorized as 'Subject ID'
                let subjectIDFound = 0;
                for ( const column in this.columnToCategoryMap ) {
                    if ( "Subject ID" === this.columnToCategoryMap[column] ) {
                        subjectIDFound += 1;
                    }
                }
                const singleSubjectIDColumn = ( 1 === subjectIDFound );
```

Obviously this code will need to be genericized for different formats, as you mention. So 'simple' categorization requires at least subject ID to be selected.

For now it makes sense to keep the text select specifically for 'Subject ID', but I have added a new issue for this future task here: https://github.com/neurobagel/annotation_tool/issues/167

Reply:
Thanks for the issue. Just to be clear: my comments refer to the `[data-cy='column-linking-table']` selector, aka the thing on the right. There I suggest not selecting based on text because that's brittle. We expect there to be a column called `"participant_id"`, but what if we end up using an example that calls this `"subject_number"`. We wouldn't want the test to fail because of that. 

So my point is: let's just pick any column on the right (e.g. the first one) because at this point our app doesn't care if we picked the right one as long as one is picked.